crates/primitives/starknet/src/tests/transaction.rs

```diff
@@ -373,3 +376,72 @@ fn test_event_wrapper_builder_with_event_content() {
 
     pretty_assertions::assert_eq!(event_wrapper, expected_event);
 }
+
+#[test]
+fn test_try_into_deploy_account_transaction() {
+    fn helper(
+        array_size: usize,
+        calldata_size: usize,
+    ) -> Result<(DeployAccountTransaction, DeployAccountTransaction), BroadcastedTransactionConversionErrorWrapper>
+    {
+        let signature: Vec<FieldElement> = vec![FieldElement::default(); array_size];
+        let constructor_calldata: Vec<FieldElement> = vec![FieldElement::default(); calldata_size];
+
+        let input = BroadcastedDeployAccountTransaction {
+            signature,
+            constructor_calldata,
+            // `FieldElement` can be trivially converted to `Felt252Wrapper` so no need to test them
+            max_fee: FieldElement::default(),
+            nonce: FieldElement::default(),
+            contract_address_salt: FieldElement::default(),
+            class_hash: FieldElement::default(),
+        };
+
+        let output: DeployAccountTransaction = input.try_into()?;
+
+        let expected_signature = bounded_vec![Felt252Wrapper::default(); array_size];
+        let expected_calldata = bounded_vec![Felt252Wrapper::default(); calldata_size];
+
+        let expect_output = DeployAccountTransaction {
+            version: 1_u8,
+            calldata: expected_calldata,
+            signature: expected_signature,
+            nonce: Felt252Wrapper::from(FieldElement::default()),
+            salt: Felt252Wrapper::from(FieldElement::default()),
+            account_class_hash: Felt252Wrapper::from(FieldElement::default()),
```

Comment:
can't we just do `FieldElement::default().into()` ?

Reply:
that works too, i just like to be more explicit about the type being used. i can change it if you want.

Reply:
yes please